src/activities/droftware_activity/Droftware_activity.qml

```diff
@@ -0,0 +1,628 @@
+                            drag.target: parent
+                            onReleased: {
+                                parent.Drag.drop()
+                                parent.x = 70
+                                parent.y = 20
+                        }
+                    }
+                }
+                Rectangle {
+                    id: brushOrange
+                        x:130
+                        y:20
+                        width: 50
+                        height: 50
+                        radius: width/2
+                        color: "orange"
+
+                        Drag.active: dragAreaOrange.drag.active
+                        Drag.hotSpot.x: 10
+                        Drag.hotSpot.y: 10
+
+                        MouseArea {
+                            id: dragAreaOrange
+                            anchors.fill: parent
+                            drag.target: parent
+                            onReleased: {
+                                parent.Drag.drop()
+                                parent.x = 130
+                                parent.y = 20
+                        }
+                    }
+                }
+                Rectangle {
+                    id: brushBlue
+                        x:190
+                        y:20
+                        width: 50
+                        height: 50
+                        radius: width/2
+                        color: "blue"
+
+                        Drag.active: dragAreaBlue.drag.active
+                        Drag.hotSpot.x: 10
+                        Drag.hotSpot.y: 10
+
+                        MouseArea {
+                            id: dragAreaBlue
+                            anchors.fill: parent
+                            drag.target: parent
+                            onReleased: {
+                                parent.Drag.drop()
+                                parent.x = 190
+                                parent.y = 20
+                        }
+                    }
+                }
+                Rectangle {
+                    id: brushPurple
+                        x:250
+                        y:20
+                        width: 50
+                        height: 50
+                        radius: width/2
+                        color: "purple"
+
+                        Drag.active: dragAreaPurple.drag.active
+                        Drag.hotSpot.x: 10
+                        Drag.hotSpot.y: 10
+
+                        MouseArea {
+                            id: dragAreaPurple
+                            anchors.fill: parent
+                            drag.target: parent
+                            onReleased: {
+                                parent.Drag.drop()
+                                parent.x = 250
+                                parent.y = 20
+                        }
+                    }
+                }
+
+                Component.onCompleted:{
```

Comment:
If we go forward each level should be better described as a dataset in QML or json.